app/helpers/application_helper/button/miq_request_approval.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::MiqRequestApproval < ApplicationHelper::Button::MiqRequest
+  needs('@miq_request', '@showtype')
```

Comment:
I thought we would have
```
needs :miq_request, :show_type
```

Work with symbol rather than string. Similar to e.g. Ruby's `attr_reader`